### ISSUE TITLE
"Copied to Clipboard" stays too long on screen

### ISSUE THREAD

**Issue by allendema** [vector-im/riotX-android#1498]
## Infos
Version 0.22.0 on Android 10

## Issue
Copying text from someone will trigger the "copied in clipboard" message which stays on screen for too long, not allowing to paste or text faster.

**Comment by bmarty** [comment]
I agree it's a bit long. Note that you can dismiss the toast with a swipe.

**Comment by TR-SLimey** [comment]
@bmarty It may be a good idea to just use a toast instead. It doesn't prevent any clicking but is just as informative and more consistent with the "Link copied to clipboard" message when long-pressing a link.

**Comment by bmarty** [comment]
Toasts cannot be dismiss and are a bit ugly, not sure it will be better. But +1 for inconsistency with "Link copied to clipboard"

**Comment by Erik-D-Mueller** [comment]
If you found your way here because the "Copied to Clipboard" messages hangs on your screen for way way too long, go to Settings > Accessability > Advanced Setting > Time to take Action

**Comment by benjaminsergent-postscript** [comment]
> If you found your way here because the "Copied to Clipboard" messages hangs on your screen for way way too long, go to Settings > Accessability > Advanced Setting > Time to take Action

Dear god, thank you. That was driving me insane. I have no idea how that got set to two minutes nor why anyone capable of using a phone at all would ever need a full two minutes to read "copied to clipboard"
